src/stream/consumption.ts

```diff
@@ -133,4 +134,86 @@ export class StreamConsumption<T, E> extends StreamBase {
 
         return s;
     }
+
+    /**
+     * Produce a readable node stream with the values from the stream.
+     *
+     * @param kind - What kind of readable stream to produce. When "raw" only strings and buffers can be emitted on the stream. Use "object" to preserve
+     * objects in the readable stream. Note that object values are not serialised, they are emitted as objects.
+     * @param options - Options for configuring how atoms are output on the stream
+     *
+     * @see {@link Stream#serialize} if the stream values should be serialized to json
+     * @group Consumption
+     */
+    toReadable(kind: "raw" | "object", options?: { atoms?: boolean }): Readable;
+
+    /**
+     * Produce a readable node stream with the raw values from the stream.
+     *
+     * @param options.single - Whether to emit only the first atom
+     *
+     * @see {@link Stream#serialize} if the stream values should be serialized to json
+     * @group Consumption
+     */
+    toReadable(kind: "raw"): Readable;
+
+    /**
+     * Produce a readable node stream in object mode with the values from the stream
+     *
+     * @param options.atoms - By default, only `ok` values are emitted, however enabling this
+     * will emit all values.
+     *
+     * @note When not using `options.atoms`, any `null` atom values will be skipped when piping to the readable stream
+     * @see {@link Stream#serialize} if the stream values should be serialized to json
+     * @group Consumption
+     */
+    toReadable(kind: "object", options?: { atoms?: boolean }): Readable;
+
+    toReadable(
+        kind: "raw" | "object",
+        options: { single?: boolean; atoms?: boolean } = {},
+    ): Readable {
```

Comment:
```suggestion
    /**
     * Produce a readable node stream with the raw values from the stream.
     *
     * @see {@link Stream#serialize} if the stream values should be serialized to json
     * @group Consumption
     */
    toReadable(kind: "raw"): Readable;

    /**
     * Produce a readable node stream in object mode with the values from the stream
     *
     * @param options.atoms - By default, only `ok` values are emitted, however enabling this
     * will emit all values.
     *
     * @note When not using `options.atoms`, any `null` atom values will be skipped when piping to the readable stream
     * @see {@link Stream#serialize} if the stream values should be serialized to json
     * @group Consumption
     */
    toReadable(kind: "object", options?: { atoms?: boolean }): Readable;
    
    /**
     * Produce a readable node stream with the values from the stream.
     *
     * @param kind - What kind of readable stream to produce. When "raw" only strings and buffers can be emitted on the stream. Use "object" to preserve
     * objects in the readable stream. Note that object values are not serialised, they are emitted as objects.
     * @param options - Options for configuring how atoms are output on the stream
     *
     * @see {@link Stream#serialize} if the stream values should be serialized to json
     * @group Consumption
     */
    toReadable(
        kind: "raw" | "object",
        options?: { atoms?: boolean },
    ): Readable {
```

Just a fix up to the overloaded method signature, no need for it to be duplicated. Run it through your LSP to make sure I didn't break anything.

Reply:
I think this was the change I made for the generated docs to work properly

Reply:
Yeah, with that setup it uses the "raw" version as the base overload for the purpose of LSP descriptions and in the generated docs. So for example, this is the first mention of `.toReadable()` in the docs
<img width="646" alt="image" src="https://github.com/clear/windpipe/assets/16392483/24e2651d-4aa5-4225-b78d-661674c5e94e">